src/Microsoft.AspNetCore.Session/DistributedSession.cs

```diff
@@ -194,58 +198,68 @@ private void Load()
         }
 
         // This will throw if called directly and a failure occurs. The user is expected to handle the failures.
-        public async Task LoadAsync()
+        public async Task LoadAsync(CancellationToken cancellationToken)
```

Comment:
Good point, these should have a default(CancellationToken) like we do elsewhere in our stack. It's one of the few things we have a default value for.